inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/Constants.java

```diff
@@ -187,7 +190,21 @@ public final class Constants {
                     .noDefaultValue()
                     .withDescription(
                             "The format of multiple sink, it represents the real format of the raw binary data");
-
+    public static final ConfigOption<String> PATTERN_PARTITION_MAP =
+            ConfigOptions.key("pattern.partition.map")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription(
+                            "Pattern rules and partition map string, " +
+                                    "eg: databasePattern1&tablePattern1:partition1,"
+                                    + "databasePattern2&tablePattern2:partition2,"
+                                    + "DEFAULT_PARTITION:partition3");
+    public static final ConfigOption<Map<String, String>> DATASOURCE_PARTITION_MAP =
```

Comment:
It is recommended not to use the format of map because it is more troublesome to configure and not easy to understand, you can use the existing format such as 'key1=value1&key2=value2...'

Reply:
The current format can be automatically converted into a map, eg:

ConfigOptions.key("datasource.partition.map").mapType()